README.md

```diff
@@ -1 +1,13 @@
 # Labs Applicant Portal
+
+# SSL Setup in Local Development
+In order for cookies to work across both the client and the server during local development, we must have SSL setup for our development environment. Here's how to set that up:
+
+1. Follow this step-by-step guide: https://www.freecodecamp.org/news/how-to-get-https-working-on-your-local-development-environment-in-5-minutes-7af615770eec/. It will generate some files.
+
+2. Grab two files from step 1, `server.key` and `server.crt`. Copy and paste both files into BOTH `ssl/` folders under /client and /server.
+
+3. Run the server as you normally would. With SSL, the ember app should be run as `yarn run start:ssl`.
```

Comment:
For future feature/developer experience improvement,  I wonder if we can use mkcert and docker-compose to automatically generate the certs needed for local development
https://github.com/FiloSottile/mkcert